src/Server/ProcessMessage.idr

```diff
@@ -98,10 +98,7 @@ loadURI conf uri version = do
     | Nothing => do let msg = "Cannot find ipkg file for \{show uri}"
                     logE Server msg
                     pure $ Left msg
-  Right res <- coreLift $ File.readFile fname
-    | Left err => do let msg = "Cannot read file at \{show uri}"
-                     logE Server msg
-                     pure $ Left msg
+  res <- Core.readFile fname
```

Comment:
I'm not 100% positive about this change. `Core.Core.readFile` implicitly throws on error, however in the server we want to catch the error as early as possible to give feedback to the user, since it's not running command one-shot but continously. Therefore, I think is better to leave it as it was or add an explicit `catch`.

Reply:
Thanks! Moving back to the function from `base`